spec/core/1.0.md

```diff
@@ -458,7 +458,6 @@ The message is *not* invalid if the second argument is a value that is not permi
 
 *Rationale:* For non-trivial properties, the client might not have enough information to decide whether a given value is valid.
 
-Setting a property implies subscribing to the property.
```

Comment:
All of this paragraph, plus the snippet below it, belongs to this statement.

Reply:
Done in 862fd43